apps/cowswap-frontend/src/modules/advancedOrders/hooks/useAdvancedOrdersActions.ts

```diff
@@ -61,12 +51,7 @@ export function useAdvancedOrdersActions() {
     [updateAdvancedOrdersState],
   )
 
-  const onSwitchTokensDefault = useSwitchTokensPlaces(onSwitchTradeOverride)
-
-  const onSwitchTokens = useCallback(() => {
-    onSwitchTokensDefault()
-    resetTradeQuote()
```

Comment:
Also an excessive quote resetting, this is already handled in `useTradeQuotePolling`